spec/yaml/swiftnav/sbp/ssr.yaml

```
@@ -802,3 +802,120 @@ definitions:
+           desc: >
+             Satellite ID. Encoded following either RTCM DF068 (GPS), DF252 (Galileo),
+             or DF488 (BDS) according to the constellation.
+       - orb_radial_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Mean Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_radial_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Radial (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_along_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Along-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - orb_cross_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Standard Deviation Cross-Track (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_mu:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Mean (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+       - clock_bound_sig:
+           type: u8
+           units: m
+           desc: >
+             Clock Bound Standard Deviation (range 0-55)
+             i<=200, mean=0.0251i
+             200<i<=240, mean=5+0.5(i-200)
+             i>240, mean=25+2(i-240)
+
+ - MSG_SSR_ORBIT_CLOCK_BOUNDS:
+     id: 0x05DE
+     short_desc: Combined Orbit and Clock Bound
+     public: False
+     fields:
+       - header:
+           type: BoundsHeader
+           desc: Header of a bounds message.
+       - ssr_iod:
+           type: u8
+           desc: IOD of the SSR bound. Encoded following RTCM DF413 specification.
+       - const_id:
+           type: u8
+           desc: Constellation ID to which the SVs belong.
+       - nb_sat:
+           type: u8
+           desc: Number of satellites. Encoded following RTCM DF387 specification.
+       - orbit_clock_bounds:
+           type: array
+           fill: OrbitClockBound
+           map_by: sat_id
+           desc: Orbit and Clock Bounds per Satellite
```

Is the number of elements in this array described by the `n_sats` / `nb_sat` field above? If so then this field needs to have an extra key to relate the two fields to the encoder/decoder can properly handle the variable length array
```suggestion
       - orbit_clock_bounds:
           type: array
           fill: OrbitClockBound
           size_fn: n_sats
           map_by: sat_id
           desc: Orbit and Clock Bounds per Satellite
```